libraries/chain/include/graphene/chain/protocol/asset_ops.hpp

```diff
@@ -442,6 +442,39 @@ namespace graphene { namespace chain {
       void            validate()const;
    };
 
+   /**
+    * @brief Update issuer of an asset
+    * @ingroup operations
+    *
+    * An issuer has general administrative power of an asset and in some cases
+    * also its shares issued to individuals. Thus, changing the issuer today
+    * requires the use of a separate operation that needs to be signed by the
+    * owner authority.
+    *
+    */
+   struct asset_update_issuer_operation : public base_operation
+   {
+      struct fee_parameters_type {
+         uint64_t fee            = 20 * GRAPHENE_BLOCKCHAIN_PRECISION;
+      };
+
+      asset           fee;
+      account_id_type issuer;
+      asset_id_type   asset_to_update;
+      account_id_type new_issuer;
+      extensions_type extensions;
+
+      account_id_type fee_payer()const { return issuer; }
+      void            validate()const;
+
+      void get_required_owner_authorities( flat_set<account_id_type>& a )const
+      { a.insert( issuer ); }
+
+      void get_required_active_authorities( flat_set<account_id_type>& a )const
```

Comment:
I think this can be removed.

Reply:
We can (IMHO) remove the `get_required_active_authorities` part, but the `get_required_owner_authorities` is crucial as we only want owner key to be able to sign that transction.

Reply:
Correct, not a big deal if keep it.